VMBackup/main/handle.sh

```diff
@@ -11,6 +11,12 @@ logfolder=$(echo $postsubstr | cut -b 1-$resultstrlen)
 logfile=$logfolder'/shell.log'
 
 rc=3
+
+if [ $1 != "enable"]
+then
+	echo "The command is not enable, exiting" >> $logfile
+	exit $rc
```

Comment:
in current code flow, we are writing a status file for non enable calls as well. it will become a business logic change now.
Have you tested this scenario ?
@soumyapattnaik / @vityagi - can you please confirm ?

Reply:
I don't think we write status for non-enable commands. But, please check and confirm @suana-ms

Reply:
@vityagi 
https://github.com/Azure/azure-linux-extensions/blob/8eafcba3f785abcd3e94a2b11a154e4a6cc1f49a/VMBackup/main/handle.py#L96

https://github.com/Azure/azure-linux-extensions/blob/8eafcba3f785abcd3e94a2b11a154e4a6cc1f49a/VMBackup/main/Utils/HandlerUtil.py#L688

**do_status_report** is the method where we write to status. So we are writing to the status file.

We might overwrite the status when backup runs as sequence number will be same